client/src/components/App.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { FaBeer } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
```

Comment:
Please update method name so that it reflects _what_ is being clicked. (In the future we will very likely have more than one click handler!)

Example: `handle[Something]Click`